### PR TITLE
add package json to npm install the library

### PR DESCRIPTION
**Pull request by thesunlover**
in response to issue #7 ;)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+		"node",
+		"js",
+		"blowfish",
+		"encryption",
+		"decryption",
+		"encypher",
+		"decypher",
+		"javascript"
+	],
+	"author": "AGorolov",
```
Comment (egoroof) [inline package.json]: lol author is the other one :)

Comment (thesunlover) [reply package.json]: sorry :D

**Comment by thesunlover** [comment]
should be fixed now

**Comment by thesunlover** [comment]
hm, is agorlov and E Gorlov the same person :D ???

File: package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
 		"decypher",
 		"javascript"
 	],
-	"author": "AGorolov",
+	"author": "Artyom Egorov",
```
Comment (egoroof) [inline package.json]: lol again :) it's not me - should be `agorlov`

**Pull request merged**